go/libraries/doltcore/sqle/dprocedures/dolt_merge.go

```diff
@@ -245,15 +245,16 @@ func performMerge(ctx *sql.Context, sess *dsess.DoltSession, roots doltdb.Roots,
 		return ws, "", noConflictsOrViolations, threeWayMerge, err
 	}
 
+	var commit string
 	if !noCommit {
 		author := fmt.Sprintf("%s <%s>", spec.Name, spec.Email)
-		commit, _, err := doDoltCommit(ctx, []string{"-m", msg, "--author", author})
+		commit, _, err = doDoltCommit(ctx, []string{"-m", msg, "--author", author})
 		if err != nil {
 			return ws, commit, noConflictsOrViolations, threeWayMerge, fmt.Errorf("dolt_commit failed")
 		}
 	}
 
-	return ws, "", noConflictsOrViolations, threeWayMerge, nil
+	return ws, commit, noConflictsOrViolations, threeWayMerge, nil
```

Comment:
Looks like the return statement on line 216 is what gets returned when we have a fast-forward merge. I think we still want to return the commit there, too, since our return args will already indicate if it's FF or not. We've got a test for the FF return param being true, but the results check is skipped, so I think that's another argument to do some sort of "commit hash skipping" support in the test framework.